download-data.py

```diff
@@ -424,6 +461,26 @@ def main() -> None:
             file=sys.stderr,
         )
 
+
+    ### Update symlink if requested, without downloading data, if the target directory exists
+    if args.update_symlink:
+
+        if args.test_data:
+            target = "test"
+        else:
+            target = args.release
+
+        if target == "current":
+            # find the most recent local release to use as target
+            dated_releases = (x.name for x in args.data_dir.iterdir() if x.is_dir and re.match("\d{4}-\d{2}-\d{2}$", x.name)
```

Comment:
I realized we probably want to check the date here too. (I didn't mean to send the last comment yet!)
```suggestion
            dated_releases = (
              x.name
              for x in args.data_dir.iterdir()
              if x.is_dir()
              and re.match("\d{4}-\d{2}-\d{2}$", x.name)
              and x.name <= datetime.date.today().isoformat()
            )
```